bugwarrior/services/__init__.py

```diff
@@ -62,7 +62,8 @@ def __init__(self, main_config, main_section, target):
         self.annotation_newlines = self._get_config_or_default('annotation_newlines', False, asbool)
         self.shorten = self._get_config_or_default('shorten', False, asbool)
 
-        self.default_priority = self.config.get('default_priority', 'M')
+        self.default_priority = self.config.get('default_priority')
```

Comment:
It is weird to force the priority `M` when I'd think: when no `default_priority` is given, no default priority should be assigned.